src/Razor/src/Microsoft.AspNetCore.Razor.LanguageServer/Completion/LspTagHelperCompletionService.cs

```diff
@@ -253,7 +253,7 @@ public override ElementCompletionResult GetElementCompletions(ElementCompletionC
                 var tagHelperDescriptors = kvp.Value;
 
                 if (tagHelperDescriptors.Count > 0 ||
-                    (!string.IsNullOrEmpty(prefix) && completionTagName.StartsWith(prefix, StringComparison.OrdinalIgnoreCase)))
+                    !string.IsNullOrEmpty(prefix) && completionTagName.StartsWith(prefix, StringComparison.OrdinalIgnoreCase))
```

Comment:
Nit, but personally I prefer the parentheses

Reply:
Sure, I'll switch this back.